src/server/repository/public/user/getOne.ts

```diff
@@ -54,13 +55,7 @@ export const getOne = async (
       if (!data) return null
       return {
         ...Objects.camelize(data),
-        roles:
-          data.roles[0] !== null
-            ? data.roles.map(({ props, ...role }) => ({
-                ...Objects.camelize(role),
-                props: { ...Objects.camelize(props), sections: (props as CollaboratorProps).sections },
-              }))
-            : [],
+        roles: data.roles[0] !== null ? data.roles.map(UserRoleAdapter) : [],
```

Comment:
Could the syntax be `roles: data.roles.map(UserRoleAdapter)` and null case handled by adapter?